citrination_client/models/columns/__init__.py

```diff
@@ -0,0 +1,8 @@
+from base import BaseColumn
```

Comment:
These import statements will cause tests to fail in python 3.x.

I think you need to either change them to use the full path (e.g. `from citrination_client.models.columns.base import BaseColumn`), or use the relative module path syntax (e.g. `from .base import BaseColumn`)

Reply:
Thanks, yup. Will replace with full paths